<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoListDialog</class>
 <widget class="QDialog" name="VideoListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="back_pushButton">
       <property name="text">
        <string>뒤로가기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_pushButton">
       <property name="text">
        <string>재생</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
